lib/ui/playground/playground_button_present_alert.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:farmsmart_flutter/ui/common/AlertWidget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class PlaygroundButtonPresentAlert extends StatelessWidget {
+  PlaygroundButtonPresentAlert({Key key, this.child});
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: RaisedButton(
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+        child: Text("Press to Show Alert"),
+        onPressed: () => _onTap(context),
+      ),
+    );
+  }
+  _onTap(BuildContext context) {
+    return showDialog<bool>(
```

Comment:
why don't we add this to the alert as a static helper.
So I can go Alert.show(viewModel: vm)